diff --git a/src/libANGLE/validationES31.cpp b/src/libANGLE/validationES31.cpp
index 6fea996..afd4855 100644
--- a/src/libANGLE/validationES31.cpp
+++ b/src/libANGLE/validationES31.cpp
@@ -1448,13 +1448,13 @@ bool ValidateBindImageTexture(const Context *context,
             return false;
         }
 
-        if (!tex->getImmutableFormat() && tex->getType() != gl::TextureType::Buffer)
+/*        if (!tex->getImmutableFormat() && tex->getType() != gl::TextureType::Buffer)
         {
             ANGLE_VALIDATION_ERROR(GL_INVALID_OPERATION,
                                    kTextureIsNeitherImmutableNorTextureBuffer);
             return false;
         }
-
+*/
         if (context->getExtensions().textureStorageCompressionEXT &&
             tex->getType() != gl::TextureType::Buffer)
         {
