diff --git a/src/libANGLE/renderer/vulkan/vk_renderer.cpp b/src/libANGLE/renderer/vulkan/vk_renderer.cpp
index 2c036af..ab0bfec 100644
--- a/src/libANGLE/renderer/vulkan/vk_renderer.cpp
+++ b/src/libANGLE/renderer/vulkan/vk_renderer.cpp
@@ -4984,10 +4984,10 @@ gl::Version Renderer::getMaxSupportedESVersion() const
     // Limit to ES2.0 if there are any blockers for 3.0.
 
     // VK_EXT_provoking_vertex is required for flat shading.
-    if (!mFeatures.provokingVertex.enabled)
+/*    if (!mFeatures.provokingVertex.enabled)
     {
         maxVersion = LimitVersionTo(maxVersion, {2, 0});
-    }
+    }*/
 
     // Multisample textures (ES3.1) and multisample renderbuffers (ES3.0) require the Vulkan driver
     // to support the standard sample locations (in order to pass dEQP tests that check these
