diff --git a/src/compiler/translator/ParseContext.cpp b/src/compiler/translator/ParseContext.cpp
index 7cac3f7..78f6cf9 100644
--- a/src/compiler/translator/ParseContext.cpp
+++ b/src/compiler/translator/ParseContext.cpp
@@ -2274,7 +2274,7 @@ void TParseContext::nonEmptyDeclarationErrorCheck(const TPublicType &publicType,
         }
 
         // GLSL ES 3.10 Revision 4, 4.9 Memory Access Qualifiers
-        switch (layoutQualifier.imageInternalFormat)
+        /*switch (layoutQualifier.imageInternalFormat)
         {
             case EiifR32F:
             case EiifR32I:
@@ -2289,7 +2289,7 @@ void TParseContext::nonEmptyDeclarationErrorCheck(const TPublicType &publicType,
                     return;
                 }
                 break;
-        }
+        }*/
     }
     else if (IsPixelLocal(publicType.getBasicType()))
     {
